usecases/monitoring/prometheus.go

```diff
@@ -141,10 +145,22 @@ type PrometheusMetrics struct {
 	T2VRequestsPerBatch   *prometheus.HistogramVec
 }
 
+// NewServerMetrics return the ServerMetrics that can be used in any of the grpc or http servers.
+//
+// Example for HTTP servers.
+//
+//	weaviate_request_duration_seconds_sum{instance="node-1", method="DELETE", route="/v1/schema/{className}", status_code="200"}
+//
+// Example For GRPC servers
+//
+//	weaviate_request_duration_seconds_sum{instance="node-1", method="gRPC", route="/grpc.health.v1.Health/Check", status_code="OK"}
 func NewServerMetrics(namespace string, reg prometheus.Registerer) *ServerMetrics {
 	r := promauto.With(reg)
 
 	return &ServerMetrics{
+		// TODO(kavi): TCPActiveConnections is not currently not used in those servers
```

Comment:
As this metric is not even reported I would vote for simply removing it to reduce the amount of dead code. 

You would not like to get TCP connection count from arbitrary go code anyway and rather check system metrics which will be 100% true.